docs/oapserver-configuration.md

```diff
@@ -0,0 +1,279 @@
+## OAPSever Configuration Introduction
+
+To configure the OAP Sever, we propose two CRDs: 
+
+- OAPServerConfig: The CRD holds all static configuration, including [environment variable](https://skywalking.apache.org/docs/main/latest/en/setup/backend/configuration-vocabulary/) and [file configuration](https://github.com/apache/skywalking/tree/master/oap-server/server-starter/src/main/resources).
+- OAPServerDynamicConfig: The CRD holds all [dynamic configuration](https://skywalking.apache.org/docs/main/latest/en/setup/backend/dynamic-config/).
+
+
+
+## Spec of OAPServerConfig
+
+| Field Name | Description                                                  |
+| ---------- | ------------------------------------------------------------ |
+| Version    | The version of OAP server, the default value is 9.0.0        |
+| Env        | The environment variable of OAP server                       |
+| File       | The static file in OAP Server, which contains three fields`file.path`、`file.name` and `file.data`.  The `file.path` plus the `file.name`  is the real file that needs to be replaced in the container image, and the `file.data` is the final data in the specific file. |
+
+
+
+## Status of OAPServerConfig 
+
+| Field Name            | Description                                          |
+| --------------------- | ---------------------------------------------------- |
+| ExpectedConfiguredNum | The number of oapserver that need to be configured   |
+| RealConfiguredNum     | The number of oapserver that configured successfully |
```

Comment:
Could you be a little more specific about these two fields? These two are read a little strange. What is `need to be configured`?

Reply:
They represents the replica number set up by the config. It's better to polish the names. How about following `Deployment`'s convention?

* `ExpectedConfiguredNum ` -> `Desired`
* `RealConfiguredNum ` -> `Ready`

Reply:
These make more sense.

Reply:
Make sense to me, thanks for the correction.

Reply:
Please update relevant documents accordingly.